plugins/aws_sts/app/views/samson_aws_sts/_fields.html.erb

```diff
@@ -0,0 +1,32 @@
+<fieldset>
+  <legend>AWS STS Tokens</legend>
+  <div class="col-lg-offset-2">
+    <div class="form-group">
+      <div class="col-lg-6">
+        <p>AWS IAM Role ARN that should be assumed by the stage:</p>
+        <%= form.text_field :aws_sts_iam_role_arn, class: "form-control", placeholder: "e.g. arn:aws:iam::123456789012:role/S3Access" %>
+        <span class="help-block">
+          This will create the following environment variables which can be used by stage commands:
+          <ul>
+            <li>STS_AWS_ACCESS_KEY_ID</li>
+            <li>STS_AWS_SECRET_ACCESS_KEY</li>
+            <li>STS_AWS_SESSION_TOKEN</li>
+          </ul>
+        </span>
+      </div>
+    </div>
+    <div class="form-group">
+      <div class="col-lg-6">
+        <p>The duration, in seconds, that the credentials should remain valid:</p>
+        <%= form.number_field :aws_sts_iam_role_session_duration, class: "form-control", placeholder: 900 %>
+        <span class="help-block">
+          Optional. Defaults to 900.
+          <ul>
+            <li>Min: 900 (15 minutes)</li>
```

Comment:
use the constants from the plugin here to keep it in sync `"#{min} (#{min / 60} minutes)"`